sections/introduction.md

```diff
@@ -0,0 +1,49 @@
+# Introduction
```

Comment:
I really like this section :) Really clear

Reply:
🙏